WordPress/playground-tools
# Prepare Playground plugin for publishing

## What?

Updates the readme to match WordPress.org expectations and adds content export filters to allow skipping of sensitive or unused files. 

## Why?

We want to publish this plugin to WordPress.org.

## How?

By updating the readme and adding filters to the data exporter.

## Testing Instructions


1. Check out the branch.
2. Add the Playground plugin to a WordPress site
3. Follow the Usage instructions in readme.txt
